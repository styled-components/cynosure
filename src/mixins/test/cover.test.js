// @flow
import cover from '../cover'

describe('cover', () => {
  it('should cover full screen when passed no parameters', () => {
    expect(cover()).toMatchSnapshot()
  })

  it('should cover full screen with an offset when passed one', () => {
    expect(cover('100px')).toMatchSnapshot()
  })
})
