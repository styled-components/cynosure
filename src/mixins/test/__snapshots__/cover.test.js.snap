// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`cover should cover full screen when passed no parameters 1`] = `
Object {
  "bottom": 0,
  "left": 0,
  "position": "absolute",
  "right": 0,
  "top": 0,
}
`;

exports[`cover should cover full screen with an offset when passed one 1`] = `
Object {
  "bottom": "100px",
  "left": "100px",
  "position": "absolute",
  "right": "100px",
  "top": "100px",
}
`;
